gigascience/gigadb-website
# All pages use the TitleBreadcrumbs component

# Pull request for issue: #1523

This is a pull request for the following functionalities:

* All pages with navigation breadcrumbs use the `TitleBreadcrumb` component
* Consistent breadcrumb labels for admin pages

## How to test?

All breadcrumbs in the code should be rendered with the `TitleBreadcrumb` component. A simple way to check this is to `grep` for `class="breadcrumb` in the project `protected/**/*.php` files

* You will notice that 3 files were not updated, because the use case for these breadcrumbs is slighlty different as it is not used page title

## How have functionalities been implemented?

* All instances of breadcrumbs were updated to use the custom component
* Breadcrumbs labels for admin pages were updated to be consistent following this scheme: `Admin / Manage / View | Create | Update`
* All component instances were updated to use the shorter path / alias: `$this->widget('TitleBreadcrumb', ...`

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
